### DISCUSSION TITLE
theme() function in arbitrary values

### DISCUSSION THREAD

**Discussion by hrishikesh-k** [tailwindlabs/tailwindcss#7100] [Help] [answered]
Hello!

I've using arbitrary values for a few of my elements now, and I was wondering if I could use the `theme()` function in there that we could use in the CSS files.

For example, I've applied `h-14` to the `<header>`. Now, I want the height of the `<section>` to be viewport height minus the height of the header. I would do something like this in the CSS:

```css
section {
  height: calc(theme('height.screen') - theme('height.14'));
}
```

This was useful as I could then update my theme's configuration and the values would correctly update.

Now with Tailwind 3.x, I'm doing something like:

```html
<section class = "h-[calc(100vh_-_3.5rem)]"></section>
```

The good part about this is that, it works, but now I'll have to make sure to update this style in case I were to change the value of `h-14`. Thankfully, since I'm using these values as a one-off thing so as to not overdo this, the occurrences that I need to update are fairly low. But I was wondering if the CSS `theme()` function is/will be supported?

Thank you for any inputs.

**Comment by adamwathan** [comment, marked as answer]
Hey! Yep this already works:

https://play.tailwindcss.com/7lFePJDDEP

**Reply by hrishikesh-k** [reply]
Oh you're right, after close inspection I saw that I was missing a closing `)` because of which this wasn't working! I'm sorry for the noise. Thank you so much!

**Reply by hrishikesh-k** [reply]
Hey, I just dived into this further, and looks like it won't work with the following config:

```html
<div class="bg-black h-[theme(height[3.5])]"></div>
```

I tried to look more and found this: https://stackoverflow.com/questions/70741264/tailwind-3-width-calc-issues-with-themewidth-1-3/70746336.

Is this a known issue or a limitation?
